spring-integration-file/src/main/java/org/springframework/integration/file/FileWritingMessageHandler.java

```diff
@@ -967,11 +969,13 @@ public void flushIfNeeded(FlushPredicate flushPredicate) {
 	 * @param filterMessage an optional message passed into the predicate.
 	 * @since 4.3
 	 */
-	public void flushIfNeeded(MessageFlushPredicate flushPredicate, Message<?> filterMessage) {
+	public void flushIfNeeded(MessageFlushPredicate flushPredicate, @Nullable Message<?> filterMessage) {
 		doFlush(findFilesToFlush(flushPredicate, filterMessage));
 	}
 
-	private Map<String, FileState> findFilesToFlush(MessageFlushPredicate flushPredicate, Message<?> filterMessage) {
+	private Map<String, FileState> findFilesToFlush(MessageFlushPredicate flushPredicate,
+			@Nullable Message<?> filterMessage) {
+
```

Comment:
Remove the extra space.

Reply:
No. That's my preference: if method head is multi-line, then we need extra blank line before method body.
It makes the code much easier to read.